MainWindow.xaml.cs

```diff
@@ -39,6 +40,7 @@ public partial class MainWindow : Window
 
         public MainWindow()
         {
+            I18n.Instance.ApplyLanguage(new CultureInfo("zh-CN"));
```

Comment:
Considering this is supposed to end up in main, I'm pretty sure we should be passing `CultureInfo.CurrentCulture` here.

Reply:
I am doing something similar to this project and it has multi-language support https://github.com/shadesofdeath/Twindows

Reply:
My bad. I added this code just for testing. I removed this and i18n instance will use  CultureInfo.CurrentCulture by default.